mathjax3-ts/output/svg/Wrappers/mtext.ts

```diff
@@ -0,0 +1,40 @@
+/*************************************************************
+ *
+ *  Copyright (c) 2019 The MathJax Consortium
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/**
+ * @fileoverview  Implements the SVGmtext wrapper for the MmlMtext object
+ *
+ * @author dpvc@mathjax.org (Davide Cervone)
+ */
+
+import {SVGWrapper, SVGConstructor} from '../Wrapper.js';
+import {CommonMtext, CommonMtextMixin} from '../../common/Wrappers/mtext.js';
```

Comment:
`CommonMtext` does not seem to be used.

Reply:
Ok, removed it.  It turns out that many of the wrappers have similar unused includes.  Should I make a PR to remove them as well?